src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -80,9 +79,13 @@ internal class LanguageWorkerChannel : ILanguageWorkerChannel, IDisposable
                 .Where(msg => msg.WorkerId == _workerId);
 
             _eventSubscriptions.Add(_inboundWorkerEvents
-                .Where(msg => msg.MessageType == MsgType.RpcLog)
+                .Where(msg => msg.MessageType == MsgType.RpcLog && (msg.Message.RpcLog.Category == null || !msg.Message.RpcLog.Category.Equals("System", StringComparison.OrdinalIgnoreCase)))
                 .Subscribe(Log));
 
+            _eventSubscriptions.Add(_inboundWorkerEvents
+               .Where(msg => (msg.MessageType == MsgType.RpcLog) && (msg.Message.RpcLog.Category != null && msg.Message.RpcLog.Category.Equals("System", StringComparison.OrdinalIgnoreCase)))
```

Comment:
move this to separate helper method IsRpcSystemLog and reuse this method above as well. and add unit tests